### PR TITLE
[IMP] core: _search_display_name replaces _name_search

### PR DESCRIPTION
**Pull request by kmagusiak**
odoo/odoo#174967

### CODE REVIEW

**Review by rco-odoo — APPROVED** [review]

**Comment by rco-odoo** [comment]
@AntoineVDV

File: content/developer/reference/backend/orm/changelog.rst [hunk 1]
```diff
@@ -4,6 +4,12 @@
 Changelog
 =========
 
+Odoo version 18.0
+=================
+
+- Searching by name is now implemented as `_search_display_name` like all other fields `#174967 <https://github.com/odoo/odoo/pull/174967>`_.
```
Comment (AntoineVDV) [inline content/developer/reference/backend/orm/changelog.rst:10]: Nitpicking
```suggestion
- Searching by name is now implemented as `_search_display_name` like all other fields. `#174967 <https://github.com/odoo/odoo/pull/174967>`_
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+ delegate=rco-odoo

**Comment by rco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**